src/Catalog/EntryFactory.php

```diff
@@ -54,6 +54,9 @@ public static function createFromArray(array $entryArray)
 
         if (count($plurals) > 0) {
             $entry->setMsgStrPlurals($plurals);
+            if(!empty($entryArray['msgid_plural'])){
```

Comment:
Does this cover the case where `msgid_plural` does not exist? ex: `!isset($entryArray['msgid_plural'])`

Reply:
yes I think...